.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,62 @@
+name: "Static Analysis"
+
+on:
+  pull_request:
+    branches:
+      - "[0-9]+.[0-9]+"
+      - "feature/*"
+    paths-ignore:
+      - "docs/**"
+  push:
+    branches:
+      - "[0-9]+.[0-9]+"
+      - "feature/*"
+    paths-ignore:
+      - "docs/**"
+
+env:
+  PHP_VERSION: "8.2"
+  DRIVER_VERSION: "mongodb/mongo-php-driver@master"
```

Comment:
Same question as in `coding-standards.yml`. Can this just use the most recent stable release?